c10/cuda/CUDAException.h

```diff
@@ -19,3 +20,13 @@
       AT_ERROR("CUDA error: ", cudaGetErrorString(__err)); \
     }                                                      \
   } while (0)
+
+// For CUDA Driver API
+#define C10_CUDA_DRIVER_CHECK(EXPR)                                \
+  do {                                                             \
+    CUresult __err = EXPR;                                         \
+    if (__err != CUDA_SUCCESS) {                                   \
+      const char* err_str;                                         \
+      AT_ERROR("CUDA error: ", cuGetErrorString(__err, &err_str)); \
```

Comment:
This is not printing the error string, it's printing the error code of cuGetErrorString.

Move the cuGetErrorString up and use `err_str` in the AT_ERROR.